subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -1083,7 +1097,8 @@ static void isr_tx(struct lll_scan_aux *lll_aux, void *pdu_rx,
 #endif /* CONFIG_BT_CTLR_PRIVACY */
 
 	/* +/- 2us active clock jitter, +1 us hcto compensation */
-	hcto = radio_tmr_tifs_base_get() + EVENT_IFS_US + 4 + 1;
+	hcto = radio_tmr_tifs_base_get() + EVENT_IFS_US + 4 +
```

Comment:
The comment above state that we add 2us active clock jitter, but we seemingly add 4 here. Is that correct? 

And should it have been?
```suggestion
	hcto = radio_tmr_tifs_base_get() + EVENT_IFS_US + 2 * EVENT_CLOCK_JITTER_US +
```